src/lib/services/publicProjectService.ts

```diff
@@ -32,90 +31,75 @@ const parseCode = (raw: RawPublicCode): PublicCode => ({
 });
 
 export const usePublicProjects = (): UseQueryResult<PublicProjectInfo[]> => {
-  const { currentChainRecord } = useWallet();
+  const projectsApiRoute = useBaseApiRoute("projects");
```

Comment:
There are 2 things on this file.
1. Remove the throwing of an error if the route is undefined or an empty string.
2. Remove it from the enabled React Query policy.